src/game/Maps/MapPersistentStateMgr.cpp

```diff
@@ -1067,7 +1067,11 @@ void MapPersistentStateManager::InitWorldMaps()
 void MapPersistentStateManager::LoadCreatureRespawnTimes()
 {
     // remove outdated data
+#ifdef DO_SQLITE
+    CharacterDatabase.DirectExecute("DELETE FROM creature_respawn WHERE respawntime <= unixepoch('now')");
```

Comment:
`unixepoch('now')` is equivalent to `unixepoch()`, so `'now'` is redundant.

However, I wonder why `UNIX_TIMESTAMP(NOW())` in MySQL was used. If `NOW()` was added to use server's TZ, then its doesn't work this way, it still produces UTC timestamp.
![image](https://github.com/cmangos/mangos-wotlk/assets/6103913/971b937c-cc83-40d6-a0a5-869f484a6a50)

Reply:
Yeah, I know they produce the same result, but because I didn't know why UNIX_TIMESTAMP(NOW()) was used here I figured there might be some reason to it and just left it the same